src/Channel.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Created by kenshoo-linux on 14/07/16.
+ */
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+public class Channel {
+
+    private String name;
+    private String campaignId;
+    private String adgroupId;
```

Comment:
Change to camel case.
Next phase:
Implement this logic:
1. If AdGroup has the three fields: engine, type, theme
1.1. Query the DB for this entity by these field (the business/composite key)
1.1.1 If found, use adGroup.id for this
1.1.2 Otherwise, insert a new entity and use the generated MySQL ID for this field
2. Otherwise, use the adGroupId received in the file (don't check if it is already exists in our DB)